docs/guides/typescript.md

```diff
@@ -351,7 +354,7 @@ const useBearStore = create<
 >(devtools(persist((set) => ({
   bears: 0,
   increase: (by) => set((state) => ({ bears: state.bears + by })),
-})))
+}), {name: "required name option"}))
```

Comment:
Here I'd suggest running Prettier after renaming to make sure that the code style is consistent with Zustand — note that Zustand's code style doesn't use double quotes for strings and has spaces around braces.

```suggestion
}), { name: 'bearStore' }))
```